reference/lidar.md

```diff
@@ -54,14 +58,60 @@ typedef struct {
 } WbLidarPoint;
 ```
 
+%tab-end
+
+%tab "C++"
+
+```cpp
+#include <webots/Lidar.hpp>
+
+namespace webots {
+  typedef struct {
+    float x;
+    float y;
+    float z;
+    int layer_id;
+    float time;
+  } LidarPoint;
+}
+```
+
+%tab-end
+
+%tab "Python"
+
+```python
+from controller import LidarPoint
+
+class LidarPoint:
+    @property
+    x, y, z, layer_id, time
+```
+
+%tab-end
+
+%tab "Java"
+
+```java
+import com.cyberbotics.webots.controller.LidarPoint;
+
+public class LidarPoint {
+  public float getX();
+  public float getY();
+  public float getZ();
+  public float getTime();
+  public double getLayerId();
+}
+```
+
+%tab-end
+
+%end
+
```

Comment:
I believe we miss a MATLAB and a ROS tab here. If a structure/class is not available, we should at least provide some plain text explanation that document how to achieve the same in these APIs.